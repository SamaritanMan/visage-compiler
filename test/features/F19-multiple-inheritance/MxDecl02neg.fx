/*
 * Negative test for mixin declarations.
 * No mixin keyword.
 * 
 * @test/compile-error
 */

class Super {}
public class PseudoMixin {}

public class PseudoMixee extends Super, PseudoMixin  {}

