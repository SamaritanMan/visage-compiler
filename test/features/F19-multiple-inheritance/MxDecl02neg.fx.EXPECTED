test/features/F19-multiple-inheritance/MxDecl02neg.fx:11: Only one non-mixin based class allowed.
public class PseudoMixee extends Super, PseudoMixin  {}
                                        ^
1 error